apps/mobile/src/screens/NftDetailScreen/NftDetailAsset/NftDetailAssetImage.tsx

```diff
@@ -12,7 +12,6 @@ type Props = {
 
 export function NftDetailAssetImage({ imageUrl, outputDimensions, onLoad }: Props) {
   if (imageUrl.includes('svg')) {
-    console.log(imageUrl);
```

Comment:
Embarrassing. I thought I removed this in the other PR

Reply:
lint rule? we can eslint-ignore when needed